test/ImpactEvaluator.t.sol

```diff
@@ -45,14 +45,16 @@ contract ImpactEvaluatorTest is Test {
             address(this)
         );
         vm.expectRevert("Wrong round");
-        impactEvaluator.setScores(1, new address[](0), new uint[](0));
+        impactEvaluator.setScores(1, new address payable[](0), new uint[](0));
         vm.expectRevert("Addresses and scores length mismatch");
-        impactEvaluator.setScores(0, new address[](1), new uint[](0));
+        impactEvaluator.setScores(0, new address payable[](1), new uint[](0));
 
-        address[] memory addresses = new address[](1);
-        addresses[0] = address(0x1);
+        address payable[] memory addresses = new address payable[](1);
+        addresses[0] = payable(address(0x1));
         uint[] memory scores = new uint[](1);
         scores[0] = 1;
+        vm.deal(payable(address(impactEvaluator)), 100);
+        // payable(address(impactEvaluator)).transfer(100);
         impactEvaluator.setScores(0, addresses, scores);
```

Comment:
@AmeanAsad healp, this call is reverting, and I don't know why

Reply:
@juliangruber can you give me permissions for this repo? I have a fix for this. 

The reason why this is failing is that certain addresses in the EVM are reserved for pre-compiles([see here](https://www.evm.codes/precompiled?fork=shanghai)). When you try to use them in a test setting, it causes a revert. The foundry error trace is a bit cryptic, but you can see it here at the end that it says `PRECOMPILE::ecrecover()` referring that an address of a precompile was used incorrectly. 

![Screen Shot 2023-08-20 at 11 10 57 PM](https://github.com/Meridian-IE/impact-evaluator/assets/43304401/e675a48b-fa26-4530-8465-f881890a9cdb)

To avoid this, you can use `vm.addr(1)` instead.

Reply:
I've given you permissions 🙏 

How can I see the foundry error message?

Reply:
I tried using `vm.addr(1)` but probably messed up, could you please commit your fix?

Reply:
To see the error message here, you go to the CI details. 

Locally, you can see it by running `forge test -vvv`. The `-vvv` controls verbosity level

Reply:
Thank you! And also thank you for the other fixes.

Are you running a formatter? If so I would like to set it up for myself as well

Reply:
Im running the VsCode "Solidity" extension by Juan Blanco :)